docs/index.md

```diff
@@ -0,0 +1,1698 @@
+  part:
+    guid: D19BE1E0660AECFF0A9AF617540AFFB7
+    name: xk3
+    slot: 9D
+    key: ecdsa-sha2-nistp521 AAAAE2VjZHNhLXNoYTItbmlzdHA1MjEAAAAIbmlzdHA1MjEAAACFBABrFyNJvVBr80bWBE9Df/b/GOnIypNxURgD0D64Nt7iT6oF163shFWLXJ04TPPSAgSX57/8e7lohol9pSczXMQaQQGaefYZKMfUvyeXpcNsu1m47axaq/HwKpwGGW0LgQ2VZQhWDQjDPP8Yr3s/krNXoV/ArwWJT7HwHocL5y7eN4TUcQ==
+```
+
+Here is how to get the values used by KBMAPI for a given template:
+
+```javascript=
+const crypto = require('crypto');
+const fs = require('fs');
+const input = fs.readFileSync('/path/to/.ebox/tpl/name');
+// This is the template:
+input.toString();
+// => '6wwBAQECAgMBCG5pc3RwNTIxQwIAy0PHzaeP/mgIO0v51q1FOp5d23d3mo5JOyybu\nNqmVrvPm9tfDLKS0s6cL9wDJ8EA2iITO5Lfx4V32IIkF3d9M2AEEOb7Rb3lFGxbIf\ny5QJUkuYwCA3hrMQABCG5pc3RwNTIxQwIAOh9YE/LiQTHO5pJ1u1otDNv9sf+uZSm\ncvQyzPcetKVOsLfCrASNjyiaFGP+yaunciEXJFczYdec+/GoI7IT7vWkEEAUc2bIX\nfrEjdMeYuzRieT4CA3hrMgABCG5pc3RwNTIxQwMAaxcjSb1Qa/NG1gRPQ3/2/xjpy\nMqTcVEYA9A+uDbe4k+qBdet7IRVi1ydOEzz0gIEl+e//Hu5aIaJfaUnM1zEGkEEEN\nGb4eBmCuz/Cpr2F1QK/7cCA3hrMwA=\n'
+const hash = crypto.createHash('sha512');
+hash.update(input.toString());
+// And this is the hash value, used as identifier:
+hash.digest('hex')
+// => 'f85b894ed02cbb1c32ea0564ef55ee2438a86c5a4988ca257dd7c71953f349d9cf0472838099967d9ec4ca15603efad17f6ac6b3f434c9080f99d6f2041799d7'
+// Instead of the hash (or together with), we can also generate a UUID
+// using the following procedure:
+var buf = hash.digest();
+// variant:
+buf[8] = buf[8] & 0x3f | 0xa0;
+// version:
+buf[6] = buf[6] & 0x0f | 0x50;
+var hex = buf.toString('hex', 0, 16);
+var uuid = [
+    hex.substring(0, 8),
+    hex.substring(8, 12),
+    hex.substring(12, 16),
+    hex.substring(16, 20),
+    hex.substring(20, 32)
+].join('-');
+```
+
+### Recovery configurations lifecycle
+
+Recovery configurations will go through a Finite State Machine during their expected lifecycles. The following are the definitions of all the possible states for recovery configurations:
+
+* `new`: This state describes the raw parameters for the recovery configuration (mostly `template`) before the HTTP request to create the recovery configuration record in KBMAPI has been made.
+* `created`: Once the recovery configuration has been created into KBMAPI through the HTTP request to `POST /recovery_configurations`. The recovery configuration now has a unique `uuid`, the attribute `created` has been added and, additionally, the process to stage this configuration through all the Compute Nodes using EDAR has been automatically started. (TBD: Shall this really be automatic or should we make it require a explicit HTTP request, just in case we want to just stage + activate on a single CN for testing before we proceed with every CN?)
```

Comment:
Yeah, we want to keep the recovery configuration as created first, just in case we want to review it for any particular error before starting stagging across the whole CNs collection. Updated.